head/src/main/kotlin/io/qalipsis/core/head/jdbc/repository/ZoneRepository.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * QALIPSIS
+ * Copyright (C) 2025 AERIS IT Solutions GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package io.qalipsis.core.head.jdbc.repository
+
+import io.micronaut.context.annotation.Requires
+import io.micronaut.data.model.query.builder.sql.Dialect
+import io.micronaut.data.r2dbc.annotation.R2dbcRepository
+import io.micronaut.data.repository.kotlin.CoroutineCrudRepository
+import io.qalipsis.core.configuration.ExecutionEnvironments
+import io.qalipsis.core.head.jdbc.entity.ZoneEntity
+
+/**
+ * Micronaut data repository to operate with [ZoneEntity].
+ *
+ * @author Francisca Eze
+ */
+@R2dbcRepository(dialect = Dialect.POSTGRES)
+@Requires(notEnv = [ExecutionEnvironments.TRANSIENT])
+interface ZoneRepository : CoroutineCrudRepository<ZoneEntity, Long>
```

Comment:
It misses a function to list the zones that are accessible to a tenant (because the either are not connected to any tenant, or because they are connected to the required tenant)